Source/WebCore/platform/graphics/win/ImageWin.cpp

```diff
@@ -25,12 +25,20 @@
 
 #include "config.h"
 #include "Image.h"
-#include "BitmapImage.h"
 
+#include "BitmapImage.h"
 #include "SharedBuffer.h"
 
+#if !PLATFORM(WIN_CAIRO)
 // This function loads resources from WebKit
 RefPtr<WebCore::FragmentedSharedBuffer> loadResourceIntoBuffer(const char*);
+#else
+// FIXME: https://bugs.webkit.org/show_bug.cgi?id=188175
+RefPtr<WebCore::FragmentedSharedBuffer> loadResourceIntoBuffer(const char*)
+{
+    return nullptr;
```

Comment:
How many tests will fail for WinCairo WK1 by this change?

Reply:
Not sure but we will be turning off WK1 soon so not sure that we should be putting any effort on it.

Reply:
I think we should remove `loadResourceIntoBuffer` in Source/WebKit/WebProcess/win/WebProcessWin.cpp in this patch.
And, we should guard `loadResourceIntoBuffer` in Source/WebKitLegacy/win/WebKitDLL.cpp with `#if !PLATFORM(WIN_CAIRO)`.

Reply:
You added a link to <https://bugs.webkit.org/show_bug.cgi?id=188175>.
Do you plan to move icon resources into WebCore.dll?
Which directory will you use? In WebCore?
AppleWin is using static WebCore.
I think you can't move icon resources into WebCore directory. Right?

Reply:
Yea I plan on addressing this and hopefully finding a way to make it work for AppleWin and WinCairo its just not a blocker here cause its already broken in Modern WebKit on WinCairo.

Reply:
What is your motivation to make WebCore DLL? Only for the icon resource problem?

Reply:
No also you can't link a non-unified WebKit on Windows.